src/seedu/addressbook/data/person/Person.java

```diff
@@ -19,6 +21,21 @@
 
     private final Set<Tag> tags = new HashSet<>();
 
+
+    /**
+     * Comparator to compare two persons based on their names
+     * Lexicographical comparison in ascending order
+     */
+    public static Comparator<Person> personNameComparator = new Comparator<Person>() {
```

Comment:
This is one way of doing it. Another way will be to make `Person` class implement `Comparable`. To me, implementing `Comparable` will be better as I will consider the lexicographical order the natural ordering of the Person class. Implementing `Comparable` allows you to use `Collections.sort(personList)` immediately.